CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,55 @@
+# Code of Conduct
+To be a truly great community, SwiftNIO needs to welcome developers from all walks of life,
+with different backgrounds, and with a wide range of experience. A diverse and friendly
+community will have more great ideas, more unique perspectives, and produce more great
+code. We will work diligently to make the SwiftNIO community welcoming to everyone.
+
+To give clarity of what is expected of our members, SwiftNIO has adopted the code of conduct
```

Comment:
`SwiftNIO` => `SwiftPM` or `Swift Package Manager`

Reply:
@jrtibbetts OK

Reply:
Could these comments stay unresolved until the suggestions have been addressed? This would make reviewing much easier. The PR still mentions SwiftNIO and has inconsistent line formatting, but for some reasons all suggestions are marked as resolved.